app/models/katello/glue/candlepin/owner.rb

```diff
@@ -38,6 +38,32 @@ def content_access_mode
         self.owner_details['contentAccessMode']
       end
 
+      def enabled_product_content(for_repositories)
+        return [] if for_repositories && for_repositories.empty?
+        repositories = for_repositories.select([:product_id, :content_id]).to_a
+        content_ids = repositories.map(&:content_id)
```

Comment:
If you are just using content_id from here then why do you need [:product_id, :content_id]